views/support/faqbrowse.tt

```diff
@@ -0,0 +1,109 @@
+<script language='JavaScript'>
+<!--
+    function showAnswer() {
+        if (!document.getElementById) {
+            return true;
+        }
+        var a = document.getElementById('answer');
+        var r = document.getElementById('rm');
+        if (!a || !r) {
+            return true;
+        }
+        r.style.display = 'none';
+        a.style.display = 'block';
+        return false;
+    }
+//-->
+
+
+</script>
+[%- CALL dw.active_resource_group("foundation") -%]
+[%- dw.need_res({ group => "foundation"}
+        "stc/faq.css"
+        ) -%]
+
+
+
+[%- UNLESS faqs -%]
+    [%- sections.title = title -%]
+    <p><b>[% '.error.nofaq' | ml %]</b></p>
+[%- ELSE -%]
+    [%- sections.title = title -%]
+<div id='faq'>
+    [%- IF faqcatarg -%]
+        <ol>
+            [%- FOR f IN faqs -%]
+                <li><a href=[%- "#faq${f.faqid}" -%]>[% f.question %]</a></li>
+            [%- END -%]
+        </ol>
+    [%- END -%]
+    [%- FOR f IN faqs -%]
+        [%- IF faqcatarg -%]
+            <h1>
+                <a id='faq[% f.faqid %]'></a>
+                <a style='text-decoration: none;' href='faqbrowse?faqid=[% f.faqid %]'>&#x00bb; [%- f.question -%]</a>
+            </h1>
+        [%- END -%]
+        <div style='margin-left: 20px;'>
+            [%- IF f.display_summary -%]
+                <div class='summary'>[% f.summary %]</div>
+                [% UNLESS f.display_answer %]
+
+                    <div class='rm'>
+                        [%- oc = faqcatarg ? '' : "onclick='return showAnswer();'" -%]
```

Comment:
should this be `f.oc`?

Reply:
Nah, it's setting a new variable named oc, not pulling one from the faq object. That said this is only used for the expand-in-place JS which we don't appear to use.

Reply:
AH! I was reading that "=" as a comparison operator, not assignment, and it shifted the context of the whole ternary. (Part of why I kind of dislike ternaries tbh, but there's kind of nothing for it in Perl since if isn't an expression.)